src/Veldrid.ImageSharp/ImageSharpCubemapTexture.cs

```diff
@@ -0,0 +1,151 @@
+    {
+        /// <summary>
+        /// An array of images, each face of a cubemap.
+        /// Access of CubemapTextures[2][3] means face 2 with mipmap level 3
+        /// </summary>
+        public Image<Rgba32>[][] CubemapTextures { get; }
+
+        /// <summary>
+        /// The width of a cubemap texture.
+        /// </summary>
+        public uint Width => (uint)CubemapTextures[0][0].Width;
+
+        /// <summary>
+        /// The height of a cubemap texture.
+        /// </summary>
+        public uint Height => (uint)CubemapTextures[0][0].Height;
+
+        /// <summary>
+        /// The pixel format cubemap textures.
+        /// </summary>
+        public PixelFormat Format => PixelFormat.R8_G8_B8_A8_UNorm;
+
+        /// <summary>
+        /// The size of each pixel, in bytes.
+        /// </summary>
+        public uint PixelSizeInBytes => sizeof(byte) * 4;
+
+        /// <summary>
+        /// The number of levels in the mipmap chain. This is equal to the length of the Images array.
+        /// </summary>
+        public uint MipLevels => (uint)CubemapTextures[0].Length;
+
+        /// <summary>
+        /// Provides standardized access to the cubemap texture array
+        /// </summary>
+        private const int PositiveXArrayLayer = 0;
+        private const int NegativeXArrayLayer = 1;
+        private const int PostitiveYArrayLayer = 2;
+        private const int NegativeYArrayLayer = 3;
+        private const int PositiveZArrayLayer = 4;
+        private const int NegativeZArrayLayer = 5;
+
+        public ImageSharpCubemapTexture(
+            string positiveXPath,
+            string negativeXPath,
+            string positiveYPath,
+            string negativeYPath,
+            string positiveZPath,
+            string negativeZPath) : this(
+                Image.Load(positiveXPath),
+                Image.Load(negativeXPath),
+                Image.Load(positiveYPath),
+                Image.Load(negativeYPath),
+                Image.Load(positiveZPath), 
+                Image.Load(negativeZPath),
+                false) { }
+
+        public ImageSharpCubemapTexture(
+            string positiveXPath,
+            string negativeXPath,
+            string positiveYPath,
+            string negativeYPath,
+            string positiveZPath,
+            string negativeZPath,
+            bool mipmap) : this(
+                Image.Load(positiveXPath),
+                Image.Load(negativeXPath),
+                Image.Load(positiveYPath),
+                Image.Load(negativeYPath),
+                Image.Load(positiveZPath),
+                Image.Load(negativeZPath),
+                mipmap)
+        { }
+
+        public ImageSharpCubemapTexture(
+            Image<Rgba32> positiveX,
+            Image<Rgba32> negativeX,
+            Image<Rgba32> positiveY,
+            Image<Rgba32> negativeY,
+            Image<Rgba32> positiveZ,
+            Image<Rgba32> negativeZ,
+            bool mipmap = false)

```

Comment:
I think the default for this should be "true" as it is in ImageSharpTexture -- mainly for consistency. NeoDemo should pass false, since a skybox doesn't need mipmaps.